satellite/console/consoleweb/server.go

```
@@ -270,8 +290,21 @@ func (s *Server) passwordRecoveryHandler(w http.ResponseWriter, req *http.Reques
 	}
 }
 
+func (s *Server) getFromArchive(path string) string {
+	f, err := s.assets.Open(filepath.Join(path))
+	if err != nil {
+		panic(err)
```

should return an error